scrapers.js

```diff
@@ -1686,6 +1686,45 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'FRA',
+    url: 'https://raw.githubusercontent.com/opencovid19-fr/data/master/dist/chiffres-cles.csv',
+    priority: 1,
+    scraper: async function() {
+      let data = await fetch.csv(this.url);
```

Comment:
Note: For sources that have timeseries data, pass `false` as date to avoid trying to find in the cache when generating timeseries.